Library/Homebrew/software_spec.rb

```diff
@@ -327,16 +327,15 @@ def select_download_strategy(specs)
 
 class BottleSpecification
   DEFAULT_PREFIX = Homebrew::DEFAULT_PREFIX
```

Comment:
It seems this constant is no longer used? However, it may not be worth sending this through travis again to remove it right now.

Reply:
The only place that I believe this was used was `Library/Homebrew/dev-cmd/bottle.rb` as `BottleSpecification::DEFAULT_PREFIX`. I believe that was the only other usage outside this file.

Reply:
`Linuxbrew/core` is currently using the idiom `HOMEBREW_PREFIX == BottleSpecification::DEFAULT_PREFIX` where Homebrew has been using `HOMEBREW_PREFIX == "/usr/local"`. So for compatibility this constant needs to remain for now. It could possibly be moved to `compat`.

See https://github.com/Homebrew/brew/pull/4851#discussion_r216400554